src/LinkedList.CPP

```diff
@@ -0,0 +1,205 @@
+#include<stdio.h>
+#include<conio.h>
+#include<alloc.h>
+#include<stdlib.h>
+
+//malloc fdunction is generic
+//append insert at end awa first node
```

Comment:
What are these comments for exactly? Which struct, or function do they belong to?